decidim-blogs/app/helpers/decidim/blogs/admin/posts_helper.rb

```diff
@@ -33,6 +33,17 @@ def post_author_select_field(form, name, _options = {})
 
           form.select(name, select_options)
         end
+
+        def publish_data(published_at)
+          data = {}
+          if published_at > Time.current
+            data[:icon] = icon("clock", aria_label: "Not published yet", role: "img")
```

Comment:
Please add a proper translated string for the label as well as the `data-popup` attribute.